setup.cfg

```diff
@@ -23,7 +23,7 @@ install_requires =
     bqplot>=0.12.29
     bqplot-image-gl>=1.4.1
     glue-core>=1.2.1
-    glue-jupyter>=0.8.1
+    glue-jupyter>=0.10
```

Comment:
This pin must have been bumped like 2-3x today!

Reply:
Indeed!